.chloggen/transformprocessor-lists.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: processor/transform
```

Comment:
Does routing processor need an entry too?

Reply:
I could use your input on this. Technically this is a breaking change since the actual function signature of `route` has changed from taking a variadic string argument to no arguments. However, `route` is documented as not taking any arguments, so anyone passing strings to the function is relying on non-functional, undocumented behavior. We could note this in the release notes, but I felt that due to the nature of the change it would probably be more confusing than helpful. None of the other functions used in the routing processor are affected.

Reply:
@jpkrohling @kovrus I'll leave the decision up to you as Code Owners

Reply:
I'd leave it out since we documented the usage of OTTL in the routing processor as experimental.